liquibase-core/src/main/java/liquibase/snapshot/SnapshotIdService.java

```diff
@@ -18,6 +18,6 @@ private SnapshotIdService() {
     }
 
     public String generateId() {
-        return base+Integer.toString(nextId++);
+        return base+ nextId++;
```

Comment:
Please correct me if I'm wrong, but I think it's a bit more efficient doing something like: `String.format("%s%d", base, nextId);`, instead of this explicit conversion.

Reply:
@arturobernalg please, ignore my previous comment. After reviewing your `StringBuilder.append()` [PR](https://github.com/liquibase/liquibase/pull/3668). I've been reading a bit more and found StringBuilder is more efficient than `String.format()`, and also that even though String concatenation is less efficient than StringBuilder,  it's better than format. I think for this particular case we can either leave it as it is considering it's a small concatenation or in alignment with your other PR update this method making use of StringBuilder as well.